server/handlers/getSurvey.js

```diff
@@ -1,13 +1,15 @@
+
+/* eslint-ignore no-console */
 const readSurvey = require('../queries/readSurvey');
 
 const getSurvey = async (req, res) => {
   try {
     // checks if params contains key id or if params itself is just one string
-    const id = !req.params.id ? req.params : req.params.id;
-
+    const { id } = req.params
     const result = await readSurvey(id.toString());
     res.status(200).json(result);
   } catch (err) {
+    console.log(err)
```

Comment:
Should be `console.warn` but ill let you off

Reply:
thank you kind sir